### ISSUE TITLE
Workflow

### ISSUE THREAD

**Issue by Badrpas** [Badrpas/Fly-simulator#10]
Let's discuss workflow here.
Is it Git Flow? Is it something different or more sophisticated?

Anyway, when it will get clear, it should be documented for future reference.

**Comment by BondarenkoArtur** [comment]
I think git flow is fine

**Comment by pavart** [comment]
Need to agree on merge strategy.
https://help.github.com/en/articles/about-merge-methods-on-github
Personally, I prefer squash merge commits. It will keep our master clean and informative.
Any preferences/objections?

**Comment by Badrpas** [comment]
Also for accepted pull requests i would suggest use of `Rebase and merge` strategy

**Comment by Badrpas** [comment]
> Personally, I prefer squash merge commits. It will keep our master clean and informative.

What are downsides of each one?

**Comment by BondarenkoArtur** [comment]
> Personally, I prefer squash merge commits. It will keep our master clean and informative.

I offer to use squash only for merging into alpha/beta/release branches (and master ofc)
For `develop` branch, let's just use regular merge without squashing

**Comment by pavart** [comment]
I'm not sure if we even need `develop` and alpha/beta/release branches.
I think we can start by having only `master` and feature branches.
Releases can be tagged on the` master` branch.
Providing that we will have CI and unit tests coverage - that should be enough.
In case we face issues/limitations of this approach - we can switch to "full git flow".

**Comment by Badrpas** [comment]
Let's stick to @pavart's approach for now. 
Following Git Flow does not much sense in current state.

And for merge strategy - i think `squashing` is ok since we still have access to hidden commits if needed.

**Comment by BondarenkoArtur** [comment]
Agreed for squashing. Also good idea to have PR number into commit after merge

**Comment by pavart** [comment]
Any discussions yet?
Closing the issue?

**Comment by Badrpas** [comment]
Will close it after corresponding document is added.

**Comment by pavart** [comment]
In this case, please create this document.
